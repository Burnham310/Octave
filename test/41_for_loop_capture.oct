main = {;instrument=1;
    for 1~<4 with i loop i 5-i 7-i-i end
}
