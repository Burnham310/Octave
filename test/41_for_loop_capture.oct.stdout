[
{
  "freq": 130.80999755859375,
  "duration": 0.5,
  "gap": 0.5,
  "amp": 1,
  "inst": "sine"
}
,
{
  "freq": 174.6103973388672,
  "duration": 0.5,
  "gap": 0.5,
  "amp": 1,
  "inst": "sine"
}
,
{
  "freq": 246.93638610839844,
  "duration": 0.5,
  "gap": 0.5,
  "amp": 1,
  "inst": "sine"
}
,
{
  "freq": 146.82925415039062,
  "duration": 0.5,
  "gap": 0.5,
  "amp": 1,
  "inst": "sine"
}
,
{
  "freq": 164.81027221679688,
  "duration": 0.5,
  "gap": 0.5,
  "amp": 1,
  "inst": "sine"
}
,
{
  "freq": 246.93638610839844,
  "duration": 0.5,
  "gap": 0.5,
  "amp": 1,
  "inst": "sine"
}
,
{
  "freq": 164.81027221679688,
  "duration": 0.5,
  "gap": 0.5,
  "amp": 1,
  "inst": "sine"
}
,
{
  "freq": 146.82925415039062,
  "duration": 0.5,
  "gap": 0.5,
  "amp": 1,
  "inst": "sine"
}
,
{
  "freq": 246.93638610839844,
  "duration": 0.5,
  "gap": 0.5,
  "amp": 1,
  "inst": "sine"
}
,
{
  "freq": 0,
  "duration": 0,
  "gap": 0,
  "amp": 0,
  "inst": "guitar"
}

]